internal/pkg/db/mongo/data.go

```diff
@@ -95,10 +96,13 @@ func (mc *MongoClient) UpdateEvent(e models.Event) error {
 
 // Get an event by id
 func (mc *MongoClient) EventById(id string) (models.Event, error) {
-	if !bson.IsObjectIdHex(id) {
-		return models.Event{}, db.ErrInvalidObjectId
+	_, err := uuid.Parse(id)
+	if err != nil {
+		if !bson.IsObjectIdHex(id) {
+			return models.Event{}, db.ErrInvalidObjectId
+		}
 	}
-	return mc.getEvent(bson.M{"_id": bson.ObjectIdHex(id)})
+	return mc.getEvent(bson.M{"_id": id})
```

Comment:
This change now looks for event IDs in Mongo as strings, not as Object IDs as was done before. When a blackbox test adds an event with a key that looks like this:

`"_id" : ObjectId("5beb91b6deafc2c5d63afaa4")`

It will not be found. The literal equivalent that this change is looking for is this:

`"_id" : "5beb91b6deafc2c5d63afaa4"`

which is incorrect.